src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -59,6 +59,10 @@
     "NAME": "Black Tentacles",
     "DESCRIPTION": "Deal !D! damage. NL This turn, all damage you deal to other enemies is dealt to this enemy instead."
   },
+  "stsjorbsmod:Blasphemer": {
+    "NAME": "Blasphemer",
+    "DESCRIPTION": "Upon draw, add !M! *Smites into your hand"
```

Comment:
I think this should probably mention "Unplayable" and the self-exhaust effect. Maybe something like:

`Unplayable. NL When drawn, add !M! *Smites into your hand and Exhaust.`

Reply:
Asked jorbs to confirm the "unplayable" in discord (I agree with you implementing it that way based on my interpretation of the sheet, but want to double check)

Reply:
Jorbs pointed out that Deus Ex Machina is a very similar example and we should follow it. I think the behavior already follows it, but that example suggests that the text should probably be:

`"Unplayable. NL When you draw this card, add !M! *Smites to your hand and Exhaust."`